src/components/Chatbox/EditBanner.tsx

```diff
@@ -18,31 +19,30 @@ import {
 import { useEditingContext } from '../../context/EditingContext';
 import { normalizeMentions } from '../../utils/mentions';
 
-const useStyles = makeStyles((theme) => ({
-  container: {
-    display: 'flex',
-    flexDirection: 'row',
-    alignItems: 'center',
-  },
-  editContainer: {
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'left',
-    width: '100%',
-    // magic to ensure that the container does not overflow its intended space
-    minWidth: '0px',
-  },
-  oldTextLabel: {
-    color: theme.palette.primary.main,
-  },
-  editIcon: {
-    margin: theme.spacing(1),
-  },
-  oldTextPreview: {
-    whiteSpace: 'nowrap',
-    overflow: 'hidden',
-    textOverflow: 'ellipsis',
-  },
+const Container = styled(Box)({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+});
+const EditContainer = styled(Box)({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'left',
+  width: '100%',
+  // magic to ensure that the container does not overflow its intended space
+  minWidth: '0px',
+});
+const OldTextLabel = styled(Typography)(({ theme }) => ({
+  color: theme.palette.primary.main,
+}));
+const OldTextPreview = styled(Typography)({
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+});
```

Comment:
I've actually noticed `Typography` has a `noWrap` prop, maybe you can remove the whole css.

Reply:
Noice !